frontend/tests/acceptance/step_definitions/notesContext.js

```diff
@@ -0,0 +1,85 @@
+const { I } = inject();
+const { deleteJobApplication } = require('../helpers/api/job_application');
+const { addNote, deleteNote } = require('../helpers/api/notes');
+const updateJobPage = require('../pages/updateJobApplicationPage');
+const notesPage = require('../pages/notesPage');
+
+const FIELDS = notesPage.fields;
+const ELEMENTS = notesPage.elements;
+let jobID;
+
+Given('the user is in the notes page', async () => {
+  updateJobPage.elements.notes_button;
+});
+
+When('the user adds a new note with following details:', async (table) => {
+  jobID = await getJobId();
+  const note = table.parse().hashes()[0];
+  notesPage.addNote(note);
+});
+
+Then('the newly created note of title {string} and content {string} should appear', async (title, content) => {
+  within(notesPage.elements.notesContainer, () => {
+    I.see(title);
+    I.see(content);
+  });
+  await tearDown();
+});
+
+Given('the following job note exists:', async (table) => {
```

Comment:
step implementation doesn't match with defination

Reply:
do you mean feature and context is different? hard to understand the context for me. I am noob 😄